### DISCUSSION TITLE
Implement Stringer interface for boil-defined types in types package

### DISCUSSION THREAD

**Discussion by xfzhu01** [aarondl/sqlboiler#932] [Ideas]
**Background**
In some scenarios, I want to print models that boiler generated to stdout using fmt package for debug, But sometimes it throws panic because there are nil fields for some boil-defined type, such as `types.NullDecimal`, The root cause is those types do not implement Stringer interface. 
**Solution**
I think it's good practice to implement Stringer interface for boil-defined types. e.g. For `types.NullDecimal`
```
func (d *NullDecimal) String() string {
	if d == nil || d.Big == nil {
		return "<nil>"
	}
	return d.Big.String()
}
```
Does it make sense?

**Comment by aarondl** [comment]
Try using a better debug output library like go-spew. It will correctly handle introspecting the nils and will give you much better output anyway.
